spec/data-spec.js

```diff
@@ -185,6 +189,13 @@ describe("Data", () => {
 				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
 				expect(+circle.attr("cy")).to.be.closeTo(expectedCy[556][i], 1);
 			});
+
+			d3.selectAll('.c3-circles-778-889 .c3-circle').each(function (d, i) {
```

Comment:
It's not a big deal, but in this test cases, `d3.select` module is used as `d3Select`. See above import.

Test case not failed, because there're any selected nodes here, which `each` block isn't callable. billboard.js' class prefix changed from `c3-` to `bb-`
- https://github.com/naver/billboard.js/wiki/How-to-migrate-from-C3.js%3F#step-3